### DISCUSSION TITLE
Help with setting up a log drain

### DISCUSSION THREAD

**Discussion by lambo81** [vercel/vercel#8009] [Help] [answered]
For now I just need to do simple things like view realtime logs, and go back and search for logs in the past.  I tried setting up Cloudflare with Vercel (it took me in this huge loop of installing, integrating, configuring)--it wasnt easy but I get it set up for my vercel project.

However, I don't see any realtime logs in the display.  

Is there any tutorial out there for setting up a log drain?

**Comment by amyegan** [comment, marked as answer]
Hi @lambo81. Sorry for the late reply! We have a basic [log drains overview](https://vercel.com/docs/integrations/log-drains-overview) and more detailed [log drains documentation](https://vercel.com/docs/log-drains). Hope that helps!
